module/VuFind/src/VuFind/Config/AccountCapabilities.php

```diff
@@ -177,6 +177,16 @@ public function getSmsSetting()
             ? 'disabled' : 'enabled';
     }
 
+    /**
+     * Get email setting.
+     *
+     * @return bool
+     */
+    public function getEmailSetting(): bool
```

Comment:
The `get...Setting` methods tend to return strings. Should this be renamed `isEmailAvailable` for clarity?

Reply:
Makes sense. I changed it.